### PR TITLE
[TG-132] Added ability to change scenario state from scenario list

### PR DESCRIPTION
**Pull request by Todmy**
- added function to ScenarioList page which toggle state of particular scenario
- refactored SecnarioService
- added e2e tests
- refactored a little bit sh-http.service
- removed Active column from scenario list page

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/garage-it/SmartHouse-frontend/pull/41" target="_blank">ReviewNinja

File: src/app/sh-http/sh-http.service.js [hunk 1]
```diff
@@ -21,8 +21,7 @@ export default class ShHttpService {
     }
     put(url, body) {
         return this.http
-            .put(url, JSON.stringify(body), this._getOptions('Put', url))
-            .map(this._convertToJson);
+            .put(url, JSON.stringify(body), this._getOptions('Put', url));
```
Comment (kosich) [inline src/app/sh-http/sh-http.service.js]: !fix
no need to send technical http information to the stream
use .error for failing requests

Comment (Todmy) [inline src/app/sh-http/sh-http.service.js]: !fixed

File: src/app/sh-http/sh-http.service.test.js [hunk 2]
```diff
@@ -78,4 +94,20 @@ describe('ShHttpService', () => {
...
+                'Content-Type': 'application/json'
+            })
+        });
+        options = options.merge({
+            method: RequestMethod.Delete,
+            url: urlMock
+        });
+        errorResponse = true;
+        sut.delete(urlMock);
+        expect(sut._errorHandling).toHaveBeenCalled();
```
Comment (kosich) [inline src/app/sh-http/sh-http.service.test.js]: !fix
its a bad practice to check sut's function call: that might be internal be hidden from outer world. or next refactoring will split that function into two.

File: src/app/sh-http/sh-http.service.js [hunk 3]
```diff
@@ -12,29 +12,26 @@ export default class ShHttpService {
     get(url) {
         return this.http
             .get(url, this._getOptions('Get', url))
-            .map(this._convertToJson);
+            .map(this._convertToJson)
+            .catch(this._errorHandling);
```
Comment (kosich) [inline src/app/sh-http/sh-http.service.js]: !fix
please, verify: seems like all catches suppress errors

File: src/app/sh-http/sh-http.service.test.js (hunk 2 above)
Comment (Todmy) [inline src/app/sh-http/sh-http.service.test.js]: !fixed

File: src/app/sh-http/sh-http.service.js (hunk 3 above)
Comment (Todmy) [inline src/app/sh-http/sh-http.service.js]: !fixed

**Pull request merged**